pkg/k8shandler/visualization.go

```diff
@@ -286,13 +286,12 @@ func (clusterRequest *ClusterLoggingRequest) createOrUpdateKibanaDeployment(prox
 			for _, secretName := range []string{"kibana", "kibana-proxy"} {
 
 				hashKey := fmt.Sprintf("%s%s", constants.SecretHashPrefix, secretName)
-				if kibanaDeployment.Spec.Template.ObjectMeta.Annotations[hashKey] != current.Spec.Template.ObjectMeta.Annotations[hashKey] {
+				if current.Spec.Template.ObjectMeta.Annotations[hashKey] != kibanaDeployment.Spec.Template.ObjectMeta.Annotations[hashKey] {
```

Comment:
this is purely for readibility -- our other comparisons follow the pattern that `current` is on the lhs